relay-server/src/actors/spooler/sql.rs

```diff
@@ -51,6 +51,26 @@ pub fn current_size<'a>() -> Query<'a, Sqlite, SqliteArguments<'a>> {
     )
 }
 
+/// Creates the query to select only 1 record from the database.
```

Comment:
It's not a `SELECT * ...`, so we should add that to the docstrings:
```suggestion
/// Creates the query to select only 1 record's `received_at` from the database.
```